sig-contributor-experience/annual-report-2023.md

```diff
@@ -3,17 +3,65 @@
+            - [Onboarded 3 "New Membership Coordinators"](https://github.com/kubernetes/community/pull/7627), 1 active and [promoted as approver](https://github.com/kubernetes/org/pull/4719) and 2 dropped off.
+        - [Kubernetes GitHub Org Membership Audit & Cleanup](https://groups.google.com/a/kubernetes.io/g/dev/c/Psiy9Iw-xCY/m/1-LVXnqRAAAJ)
+        - [Updates to Kubernetes Org Membership Requirements](https://groups.google.com/a/kubernetes.io/g/dev/c/X6zDzaDRYaU/m/nVgDcG1rAwAJ)
+        - [Onboarded SIG Etcd to Kubernetes GitHub Organisation & Prow](https://github.com/kubernetes/community/pull/7372) (major thanks to @cblecker, @mrbobbytables)
+        - [Peribolos tool, extended to manage repo level permissions under GitHub teams](https://github.com/kubernetes/org/pull/2614)
+
+    - Events:
+        - 3 successful Kubernetes contributors summits:
+            - [Europe, hosted in Amsterdam, Netherlands](https://www.kubernetes.dev/events/2023/kcseu/)
+            - [China, hosted in Shanghai](https://www.kubernetes.dev/events/2023/kcscn/) (special thanks to @pacoxu for [leading the event](https://github.com/kubernetes/community/issues/7510))
+            - [North America, hosted in Chicago, Illinois](https://www.kubernetes.dev/events/2023/kcsna/)
+
+    - Mentoring:
+        - 1 successful mentoring cohort:
+            - [Kustomize maintainer training cohort](https://groups.google.com/a/kubernetes.io/g/dev/c/M5OphEVsv5o/m/zc6G4H15AAAJ)
+
+    - Elections:
+        - [Kubernetes Steering Committee Elections, for the 2023 cycle](https://groups.google.com/a/kubernetes.io/g/dev/c/bZpmcKA_900/m/EnlPmiDBAQAJ)
+        - facilitated [Kubernetes Code of Conduct Committee Election, for the 2023 cycle](https://groups.google.com/a/kubernetes.io/g/dev/c/YfyhGdTr3iA/m/AxT0OtIjBAAJ)
+
+    - Contributor-comms:
+        - got access to Linkedin
+        - started process to get access to kubernetes twitter account
+        - LWKD (Last Week in Kubernetes Development) group became part of contributor-comms, onboarded 3 new dedicated maintainers [1](https://github.com/kubernetes/org/pull/4519) [2](https://github.com/kubernetes/org/pull/4818)
+        - SIG spotlight blogs
+            - [Spotlight on SIG CLI](https://kubernetes.io/blog/2023/07/20/sig-cli-spotlight-2023/)
+            - [Spotlight on SIG Testing](https://kubernetes.io/blog/2023/11/24/sig-testing-spotlight-2023/)
+            - [Spotlight on SIG Architecture: Conformance](https://kubernetes.io/blog/2023/10/05/sig-architecture-conformance-spotlight-2023/)
+            - [Spotlight on SIG Architecture: Production Readiness](https://kubernetes.io/blog/2023/11/02/sig-architecture-production-readiness-spotlight-2023/)
+            - [Introducing SIG etcd](https://kubernetes.io/blog/2023/11/07/introducing-sig-etcd/)
+            - how many by new contributors?
+        - new subproject leadership roles?
```

Comment:
@kaslin, I've added a list of Spotlight blogs published in 2023. Please review/add/update.

Also the pending 2 questions:

 - how many (SIG spotlight blogs) by new contributors?
 - new subproject leadership roles in contributor-comms?